src/lambdapi.ml

```diff
@@ -95,6 +108,9 @@ let spec =
     ; ( "--confluence"
       , Arg.String (fun cmd -> confluence_checker := Some(cmd))
       , "<cmd> Runs the given confluence checker." )
+    ; ( "--termination"
```

Comment:
It should be explained somewhere in lambdapi.ml or USER_MANUAL.md what happens if there are several --confluence or --termination flags.